Formula/mpich.rb

```diff
@@ -59,6 +60,9 @@ def install
                           "--disable-silent-rules",
                           "--prefix=#{prefix}",
                           "--mandir=#{man}",
+                          # Use libfabric https://lists.mpich.org/pipermail/discuss/2021-January/006092.html
+                          "--with-device=ch4:ofi",
+                          "--with-libfabric=#{Formula["libfabric"].opt_prefix}",
```

Comment:
Yea, looking at the bottles from before you included this flag, they linked with `libfabric` just fine, so I guess the issue is something else.

Reply:
They linked against an internal libfabric. We want to used brewed libfabric instead.